### DISCUSSION TITLE
Kubernetes v1.27 Feature Blogs publication order and dates

### DISCUSSION THREAD

**Discussion by harshitasao** [kubernetes/sig-release#2208] [General]
In v1.27, we plan to publish 4 blogs per week.
[14](https://github.com/orgs/kubernetes/projects/117/views/4?filterQuery=status%3ATracked+blog-status%3A%22Ready+for+Review%22) feature blogs are Ready for Review. After analysis of the Major Themes and some balancing around SIGs, this is the proposal.

| Title | Link | Order | Date |
|-----------|------------|------------|------------|
|Respect PodTopologySpread after rolling upgrades|https://github.com/kubernetes/website/pull/39777|1|17/04/2023|
|Speed up recursive SELinux label change|https://github.com/kubernetes/website/pull/39836|2|18/04/2023|
|ReadWriteOncePod PersistentVolume Access Mode|https://github.com/kubernetes/website/pull/39817|3|20/04/2023|
|Node log query|https://github.com/kubernetes/website/pull/39829#top|4|21/04/2023|
|Server Side Unknown Field Validation & OpenAPI v3|https://github.com/kubernetes/website/pull/39844|5|24/04/2023|
|Auto-refreshing Official CVE Feed|https://github.com/kubernetes/website/pull/39644|6|25/04/2023|
|Introduce nodeExpandSecret in CSI PV source|https://github.com/kubernetes/website/pull/39834|7|27/04/2023|
|Allow StatefulSet to control start replica ordinal numbering|https://github.com/kubernetes/website/pull/37418|8|28/04/2023|
|KMS v2 Improvements|https://github.com/kubernetes/website/pull/40194|9|01/05/2023|
|Container Resource based Pod Autoscaling|https://github.com/kubernetes/website/pull/39822|10|02/05/2023|
|Auto remove PVCs created by StatefulSet|https://github.com/kubernetes/website/pull/39819|11|04/05/2023|
|Memory QOS Alpha2 |https://github.com/kubernetes/website/pull/39853|12|05/05/2023|
|VolumeGroupSnapshot|https://github.com/kubernetes/website/pull/39415|13|08/05/2023|
|ApplySet : kubectl apply --prune redesign and graduation strategy|https://github.com/kubernetes/website/pull/40070|14|09/05/2023|
|Reserve nodeport ranges for dynamic and static allocation|https://github.com/kubernetes/website/pull/40204|15|11/05/2023|
||https://github.com/kubernetes/website/pull/39846|16|12/05/2023|
||https://github.com/kubernetes/website/pull/40156|17|15/05/2023|

**Comment by sftim** [comment]
Sounds good to me.

**Comment by sftim** [comment]
Is there room to add https://github.com/kubernetes/website/pull/40204? I presume it'd go at the back of the queue.

**Comment by sftim** [comment]
Possibly https://github.com/kubernetes/website/pull/39846 as well?
